ckanext/ontario_theme/templates/internal/group/read.html

```diff
@@ -58,26 +59,26 @@
     this_group_name=this_group_name)
     }}
   {% endif %}
+  {% block page_pagination %}
+        {{ page.pager(q=q) }}
+  {% endblock page_pagination %}
+  </div>
 {% endblock packages_list %}
+{% endblock primary_content_inner %}
 
-{% block secondary %}
-  <aside class="secondary col-sm-3 group-packages filter-results">
-    {% block secondary_content %}
-      {% snippet "group/snippets/info.html", group=c.group_dict, show_nums=true %}
+{% block organization_facets %}
```

Comment:
I think we can change this back to secondary? and above filters we can add back the secondary_content block